static_frame/core/store_zip.py

```diff
@@ -3,22 +3,67 @@
 import pickle
 from io import StringIO
 from io import BytesIO
-
+from concurrent.futures import ProcessPoolExecutor
 
 from static_frame.core.frame import Frame
 from static_frame.core.store import Store
 from static_frame.core.store import store_coherent_non_write
 from static_frame.core.store import store_coherent_write
 from static_frame.core.store import StoreConfigMap
+from static_frame.core.store import StoreConfig
 from static_frame.core.store import StoreConfigMapInitializer
 from static_frame.core.util import AnyCallable
 from static_frame.core.container_util import container_to_exporter_attr
+from static_frame.core.util import DtypesSpecifier
+
+
+FrameConstructor = tp.Callable[[tp.Any], Frame]
+
+
+class HashableConfigParams(tp.NamedTuple):
+    index_depth: int
+    index_name_depth_level: tp.Optional[tp.Union[int, tp.Iterable[int]]]
+    columns_depth: int
+    columns_name_depth_level: tp.Optional[tp.Union[int, tp.Iterable[int]]]
+    columns_select: tp.Optional[tp.Iterable[str]]
+    dtypes: DtypesSpecifier
+    consolidate_blocks: bool
+
+
+class BuildFrameParams(tp.NamedTuple):
+    src: bytes
+    name: tp.Hashable
+    config: HashableConfigParams
+    constructor: FrameConstructor
+
 
 class _StoreZip(Store):
 
     _EXT: tp.FrozenSet[str] = frozenset(('.zip',))
     _EXT_CONTAINED: str = ''
 
+    @classmethod
+    def _container_type_to_constructor(cls, container_type: tp.Type[Frame]) -> FrameConstructor:
+        raise NotImplementedError
+
+    @classmethod
+    def _build_frame_explicit(cls,
```

Comment:
I am not sure "explicit" is the right word here, but I cannot come up with anything better right now.

Reply:
Maybe `_frame_from_bytes`?